### PR TITLE
Assignment 3 commit

### PR DESCRIPTION
**Pull request by simon-p-r**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -10,19 +10,24 @@ var Version = require('./version');
 var internals = {};
 
 
-internals.init = function () {
+exports.init = internals.init = function (port, callback) {
```
Comment (chyld) [inline lib/index.js]: you're not using internals. you're creating the property init, but you're not using it - just remove internals. it's not needed here.

**Comment by simon-p-r** [comment]
Ok @chyld, I have removed the internal.init on your recommendation.  Thanks for the feedback!

File: lib/index.js [hunk 2]
```diff
@@ -10,19 +10,24 @@ var Version = require('./version');
 var internals = {};
 
 
-internals.init = function () {
+exports.init = function (port, callback) {
```
Comment (EvanMarkHopkins) [inline lib/index.js]: Replace all instances of ```callback``` with ```next```, the reason behind such naming convensions are described by eran in #80 and in the [style guide](https://github.com/hapijs/contrib/blob/master/Style.md#callback)

**Comment by simon-p-r** [comment]
Ok thanks @EvanMarkHopkins

**Pull request closed without merging**